server/cmd/mmctl/commands/team_e2e_test.go

```diff
@@ -125,6 +125,51 @@ func (s *MmctlE2ETestSuite) TestDeleteTeamsCmdF() {
 		config, _, _ = c.GetConfig(context.TODO())
 		config.ServiceSettings.EnableAPITeamDeletion = &enableConfig
 		_, _, _ = c.UpdateConfig(context.TODO(), config)
+
+		_, err = s.th.App.GetTeam(teamName)
+		s.Require().NotNil(err)
+	})
+
+	s.Run("Delete team with disabled config as local client", func() {
+		printer.Clean()
+
+		teamName := "teamname" + model.NewRandomString(10)
+		teamDisplayname := "Mock Display Name"
+		cmd := &cobra.Command{}
+		cmd.Flags().String("name", teamName, "")
+		cmd.Flags().String("display-name", teamDisplayname, "")
+		err := createTeamCmdF(s.th.LocalClient, cmd, []string{})
+		s.Require().Nil(err)
+
+		cmd = &cobra.Command{}
+		args := []string{teamName}
+		cmd.Flags().String("display-name", "newDisplayName", "Team Display Name")
+		cmd.Flags().Bool("confirm", true, "")
+
+		c := s.th.LocalClient
+
+		enableConfig := false
+		config, _, _ := c.GetConfig(context.TODO())
+		holdConfig := config.ServiceSettings.EnableAPITeamDeletion
+		// Set EnableAPITeamDeletion
+		config.ServiceSettings.EnableAPITeamDeletion = &enableConfig
+		_, _, _ = c.UpdateConfig(context.TODO(), config)
+
+		// Deletion should succeed for both local and SystemAdmin client now
```

Comment:
This is just testing the local client behavior, right? Deletion should fail for sysadmins when the config is disabled as the next test proves if I understood correctly